contracts/contracts/FleekERC721.sol

```diff
@@ -380,30 +416,44 @@ contract FleekERC721 is
         }
     }
 
-    /*//////////////////////////////////////////////////////////////
-        ACCESS POINTS
-    //////////////////////////////////////////////////////////////*/
+    /**
+     * @dev Sets an address as verifier of a token.
+     * The verifier must have `CollectionRoles.Verifier` role.
+     *
+     * May emit a {MetadataUpdate} event.
+     *
+     * Requirements:
+     *
+     * - the tokenId must be minted and valid.
+     * - the sender must be the owner of the token.
+     * - the verifier must have `CollectionRoles.Verifier` role.
```

Comment:
are token verifiers selected from a list of collection verifiers? how could a NFA specify a different verifier to be used for its own token?

Reply:
The token verifiers must have the collection role Verifier to be added as a token verifier.

The Verifier collection role can:
- Set APs name and content verified flag
- They will be able also to set the verify flag in the token itself in a future PR.

The Verifier doesn't approve or reprove any APs.

Reply:
Summary of the conversation on Discord:

Outside parties are going to become verifiers in the near future so this lays the foundation for that. Such verification services are going to be bought by nfas and this is where the incentive to run a verifier truly lies. We still need to figure out the process of admitting new external verifiers to the project.